doc/api/errors.md

```diff
@@ -441,86 +441,98 @@ checks or `abort()` calls in the C++ layer.
 
 ## System Errors
 
-System errors are generated when exceptions occur within the Node.js
-runtime environment. Typically, these are operational errors that occur
-when an application violates an operating system constraint such as attempting
-to read a file that does not exist or when the user does not have sufficient
-permissions.
+Node.js generates system errors when exceptions occur within its runtime
+environment. These usually occur when an application violates an operating
+system constraint. For example, a system error will occur if an application
+attempts to read a file that does not exist.
 
-System errors are typically generated at the syscall level: an exhaustive list
-of error codes and their meanings is available by running `man 2 intro` or
-`man 3 errno` on most Unices; or [online][].
+System errors are usually generated at the syscall level. For a comprehensive
+list, see the [`errno`(3) man page][].
 
-In Node.js, system errors are represented as augmented `Error` objects with
-added properties.
+In Node.js, system errors are `Error` objects with extra properties.
 
 ### Class: SystemError
 
-#### error.info
-
-`SystemError` instances may have an additional `info` property whose
-value is an object with additional details about the error conditions.
-
-The following properties are provided:
-
+* `address` {string} If present, the address to which a network connection
+  failed
 * `code` {string} The string error code
+* `dest` {Buffer} If present, the file path destination when reporting a file
+  system error
 * `errno` {number} The system-provided error number
-* `message` {string} A system-provided human readable description of the error
+* `info` {Object} If present, extra details about the error condition
+* `message` {string} A system-provided human-readable description of the error
+* `path` {Buffer} If present, the file path when reporting a file system error
+* `port` {number} If present, the network connection port that is not available
 * `syscall` {string} The name of the system call that triggered the error
-* `path` {Buffer} When reporting a file system error, the `path` will identify
-  the file path.
-* `dest` {Buffer} When reporting a file system error, the `dest` will identify
-  the file path destination (if any).
+
+#### error.address
+
+* {string}
+
+If present, `error.address` is a string describing the address to which a
+network connection failed.
 
 #### error.code
 
 * {string}
 
-The `error.code` property is a string representing the error code, which is
-typically `E` followed by a sequence of capital letters.
+The `error.code` property is a string representing the error code. `error.code`
```

Comment:
Would be even nicer if this is linked at the top of the document instead of hiding in at `####`, but that can be done in another PR (but then this would not be properly fixing https://github.com/nodejs/node/issues/23975).

Reply:
I moved the note about code stability to the one under the class Error. Hopefully that fixes the issue.